src/modules/signer/signer.types.ts

```diff
@@ -8,9 +8,9 @@ export enum ProviderType {
 }
 
 export enum ProviderEvent {
```

Comment:
I guess these are from the wallet-connect or the metamask code? Are they documented somewhere? If so, could we add a link to where they are documented or defined?